### ISSUE TITLE
Eclipse IDE for Java Developers - Windows on ARM

### ISSUE THREAD

**Issue by daliasheasha** [eclipse-platform/eclipse.platform.releng.aggregator#597]
Eclipse IDE for Java Developers is not available for Windows on ARM devices:
![image](https://user-images.githubusercontent.com/16565819/189425647-9a0b8224-2175-4c54-9ed2-2ce80eb48cbd.png)

When I tried to run the Windows _86_64 version of Eclipse on a Surface Pro X device with the Microsoft Build of OpenJDK for ARM, I get the following error:
![image](https://user-images.githubusercontent.com/16565819/189427912-3e83b0d1-c988-4414-8255-c5b5de42e906.png)

The Java Engineering Group at Microsoft would like to help get Eclipse to run on Surface Pro X. Is there anything our team can do to help produce an Eclipse distribution that is compatible with ARM devices running Windows?

**Comment by akurtakov** [comment]
I'm closing this one as duplicate of https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/577 where actual work has happened but is blocked by resources limitation. Even if we make cross compiling working (as the guy is trying) I don't think we should throw out bits that haven't seen at least basic smoke testing. That's why I say reason to not progress further for now is missing build/test machine.
